### ISSUE TITLE
[SCREENREADER]: Text Appointments - Focus must be managed when users press Delete in the modal dialog

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2430]
## Description

When I press Delete in the update mobile phone number modal window, there is no announcement to screen readers something is happening. We should be setting focus on the heading inside the alert box so users are oriented that an action is required. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

1. As a screen reader user, I want to hear the text "Are you sure? Heading level three" read aloud when I press "Delete".
2. As a screen reader user, I want to press`DOWN_ARROW` or `Ctrl + Opt + RIGHT_ARROW` to move to the next text block in the alert box because I have been oriented properly on the page.

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)
* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-14 at 3 21 28 PM](https://user-images.githubusercontent.com/934879/66780572-c0489c80-ee96-11e9-95dc-49005cce9ea7.png)

**Comment by lkoenigsberg** [comment]
This will move to the Authenticated Experience board for consideration as we merge account and profile to a single page and redesign. (cc/ @Justin-Pickett  )

**Comment by 1Copenut** [comment]
@Samara-Strauss Was this work done and tested, or closed as not done for right now?

**Comment by bradl3yC** [comment]
@1Copenut this was tested in this PR https://github.com/department-of-veterans-affairs/vets-website/pull/11484

**Comment by 1Copenut** [comment]
Thanks @bradl3yC , I'll update the product list now.
